app/services/api/v3/participant_declarations_query.rb

```diff
@@ -69,7 +69,7 @@ def declarations_scope
 
       def previous_declarations_scope
         scope = with_joins(ParticipantDeclaration)
-          .where(participant_profile: { induction_records: { induction_programme: { partnerships: { lead_provider: } } } })
+          .where(cpd_lead_provider_id: lead_provider.id)
```

Comment:
Could we keep the old where clause as this is required to surface declarations that don't belong to the current provider, but an application that has transferred now to them (thus the induction record/lead provider lookup)?